product_variant_sale_order_route/models/product.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2020 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+from odoo import models, fields
+
+
+class Product(models.Model):
+    _inherit = 'product.product'
+
+    sale_order_line_route_id = fields.Many2one(
```

Comment:
This must be a M2m to allow different routes for the same variant.
Moreover, I'm wondering if it shouldn't be named `route_ids` in order to obfuscate `product.template`'s `route_ids`.

Reply:
After checking the customer requirements again, as we only need ONE route per variant, let's keep it simple as it is, instead of complicating things :wink: